addon/components/sequential-render.js

```diff
@@ -102,16 +101,27 @@ export default Component.extend({
     Set this to false if you only need the component to render the content.
 
     @argument asyncRender
+    @deprecated presence of getData attribute implies async operation
     @type boolean
     @public
     @default true
   */
   asyncRender: true,
 
+  /**
+    Promise that can be executed to perform the required actions.
```

Comment:
Shouldn't this be a function that returns a promise? and the type a function?

Reply:
yes, thanks for catching, have updated in recent commit